docs/user/canvas.asciidoc

```diff
@@ -51,7 +51,7 @@ To open *Canvas*, open the main menu, then click *Canvas*.
 
 To use the background colors, images, and data of your choice, start with a blank workpad.
 
-. On the *Canvas workpads* page, click *Create workpad*.
+. On the *Canvas* page, click *Create workpad*.
```

Comment:
Is "page" a term specific to Canvas? Otherwise, I don't see that term being used much in Kibana.

Reply:
Yes. We have *Canvas* and *Dashboard* pages.